VideoLocker/src/main/java/org/edx/mobile/view/adapters/StaticFragmentPagerAdapter.java

```diff
@@ -87,5 +113,27 @@ private Fragment generateFragment() {
             fragment.setArguments(args);
             return fragment;
         }
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
In `UserProfileFragmentTest`, we check whether the correct `Item`s are passed to the adapter; To see if they are equal, we need an `equals` method.